includes/class-sensei-admin.php

```diff
@@ -348,9 +348,7 @@ public function register_scripts( $hook ) {
 		Sensei()->assets->enqueue( 'sensei-event-logging', 'js/admin/event-logging.js', [ 'jquery' ], true );
 
 		// Sensei custom navigation.
-		if ( $screen && ( in_array( $screen->id, [ 'edit-course', 'edit-course-category' ], true ) ) ) {
-			Sensei()->assets->enqueue( 'sensei-admin-custom-navigation', 'js/admin/custom-navigation.js', [], true );
-		}
+		Sensei()->assets->enqueue( 'sensei-admin-custom-navigation', 'js/admin/custom-navigation.js', [], true );
```

Comment:
We want to enqueue this script only for certain pages.
https://github.com/Automattic/sensei/pull/4687#discussion_r793612017

Reply:
@merkushin  I reverted the change and added new screens: https://github.com/Automattic/sensei/pull/4692/commits/63d63f261ce0b2524ae3bb395f4b911e3bea71ba#diff-a700db88caca1edaa7ce24fc214d9344458fb9a1a636e4f8fa6bfb9eaa9a93dfR351